index.js

```diff
@@ -0,0 +1,18 @@
+const glob = require('fast-glob');
+
+const lintFile = require('./lint-file');
+const logErrors = require('./log-errors');
+
+glob('**/*', { ignore: ['node_modules'] })
+  .then(files => files.map(lintFile))
```

Comment:
Single `then` is enough, steps are synchronous. Use `pipe` if you want to keep the look of the code.

Reply:
which library do you recommend for pipes? ramda?

Reply:
For now I'd start with some single-purpose package. Like [this one](https://github.com/acvos/function-pipe). If you need more similar functions, go on with Ramda.

Reply:
done